include/rocksdb/trace_reader_writer.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
I agree with you on the confusion part. I am thinking out loud here: I think we should care more about the filenames in include/* than in other directories.